source/braille.py

```diff
@@ -1190,35 +1196,58 @@ def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):
 
 
 def _getFormattingTags(
-	field: dict[str, str], fieldCache: dict[str, str], formatConfig: dict[str, bool]
+	field: dict[str, str],
+	fieldCache: dict[str, str],
+	formatConfig: dict[str, bool],
 ) -> str | None:
+	"""Get the formatting tags for the given field and cache.
+
+	Formatting tags are calculated according to the preferences passed in :param:`formatConfig`.
```

Comment:
Is this typical sphinx formatting? I've never seen param used like this before

```suggestion
	Formatting tags are calculated according to the preferences passed in formatConfig.
```